foundation/src/main/java/org/cbio/portal/pipelines/foundation/ClinicalDataProcessor.java

```diff
@@ -44,19 +44,38 @@
  * @author Prithi Chakrapani, ochoaa
  */
 public class ClinicalDataProcessor implements ItemProcessor<CaseType, String> {
+    
+    private boolean addData;
+    private List<String> columns;
 
+    public void setProperties(boolean addData, List<String> columns) {
+        this.addData = addData;
+        this.columns = columns;
+    }
+    
     @Override
-    public String process(final CaseType caseType) throws Exception {
+    public String process(CaseType caseType) throws Exception {
         ClinicalData clinicalData = new ClinicalData(caseType);
         Map<String,String> map = clinicalData.getStagingMap();
         
+        // get standard clinical data values for record
         List<String> record = new ArrayList();
         for (String field : map.keySet()) {
             String value = clinicalData.getClass().getMethod(map.get(field)).invoke(clinicalData).toString();
             record.add(value);
         }
         
-        return StringUtils.join(record, "\t");        
+        // add non-human content data to record for if applicable
```

Comment:
non-human content is part of the new schema changes - it captures whether a patient has been exposed to certain viruses